packages/scaffold/src/modal/w3m-router/index.ts

```diff
@@ -129,6 +129,12 @@ export class W3mRouter extends LitElement {
         return html`<w3m-wallet-send-select-token-view></w3m-wallet-send-select-token-view>`
       case 'WalletSendPreview':
         return html`<w3m-wallet-send-preview-view></w3m-wallet-send-preview-view>`
+      case 'ConnectWallets':
```

Comment:
No, this will be an extra view where we show all the wallets

Connect view will be email, social, show wallets button.

Customisation will be really important now, so we should give that priority after this

Reply:
💯